src/reference/modules/installation/pages/_partials/debian_repo.adoc

```diff
@@ -21,7 +21,9 @@ You need to replace the user name (LOGIN) and the password (PASSWORD) by your Ru
 [source, Bash]
 ----
 
-echo "deb https://download.rudder.io/apt/7.3/ $(lsb_release -cs) main" > /etc/apt/sources.list.d/rudder.list
+# arch may be replaced by i386, armhf or arm64 depending on your subscription and actual architecture
+
+echo "deb [arch=amd64] https://download.rudder.io/apt/7.3/ $(lsb_release -cs) main" > /etc/apt/sources.list.d/rudder.list
```

Comment:
this will not work on arm systems

Reply:
(this time on the right line)

Reply:
there is a comment just above to explain that